### DISCUSSION TITLE
unwanted transparency

### DISCUSSION THREAD

**Discussion by bobblkabb** [hyprwm/Hyprland#2467] [Q&A]
Hello,

when I make an object transparent in OpenSCAD, the wallpaper shines through. This is, depending on the wallpaper, confusing. Is it possible to set this so that the background does not show through in OpenSCAD?
Played with opacity rule but didn't help.

Michael

**Comment by vaxerski** [comment]
forcergbx rule

**Comment by bobblkabb** [comment]
Doesn't work. Transparency still shows the wallpaper. To be clear: i want opacity in OpenSCAD but I don't want to see the desktop background.
![screenshot](https://github.com/hyprwm/Hyprland/assets/7815101/cbb9fb86-40af-4625-9b05-71ff3c33da7a)

Edit: 
windowrulev2=forcergbx,class:org.openscad.openscad
